projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -18,7 +21,23 @@ export default function SubscribePanels() {
 		getSubscriberCount( count => setSubscriberCount( count ) );
 	}, [] );
 
-	return ! Number.isFinite( subscriberCount ) || subscriberCount <= 0 ? null : (
+	// Only show this for posts for now (subscriptions are only available on posts).
+	const postType = useSelect( select => select( editorStore ).getCurrentPostType(), [] );
+	if ( 'post' !== postType ) {
+		return null;
+	}
+
+	// Subscriptions will not be triggered on WoA's private sites.
+	if ( isAtomicSite() && isPrivateSite() ) {
```

Comment:
Can private sites have subscribers / collaborators?

Reply:
They can have subscribers, yes, if they were public at one point and then switched to private, but subscription emails are not being sent once the site is private.

Reply:
But they might be getting notifications? What about members of the site that are following / getting emails for the site?

Reply:
No emails will be sent to anyone, and no post will appear in the Reader, whether you're a member of the site or not.